docs/docs/quickstart.mdx

```diff
@@ -40,6 +40,7 @@ container images and will load up some examples. Once all containers
 are downloaded and the output settles, you're ready to log in.
 
 ⚠️ If you get an error message like `validating superset\docker-compose-image-tag.yml: services.superset-worker-beat.env_file.0 must be a string`, you need to update your version of `docker-compose`.
+Note that `docker-compose` is on the path to deprecation and you should now use `docker compose` instead.
```

Comment:
I think instead of adding this line, you could just replace `docker-compose` in the preceding line with `docker compose`